### PR TITLE
Fake implementation of telajax

### PR DESCRIPTION
**Pull request by nicoTolly**
This PR adds a fake implementation of telajax that allows the code to be compiled on every architecture. Mppa syntax checking has been added in travis.yaml too.

### CODE REVIEW

File: backend/mppa/src/context.rs [hunk 1]
```diff
@@ -1,4 +1,6 @@
 //! MPPA evaluation context.
+#[cfg(not(feature = "real_mppa"))]
+use crate::fake_telajax as telajax;
```
Comment (Elarnon) [inline backend/mppa/src/context.rs]: I would put this one close to the real `use telajax`

File: backend/mppa/src/printer.rs [hunk 2]
```diff
@@ -222,8 +222,8 @@ impl MppaPrinter {
         for i in 0..n {
             let start = format!("d{}", i);
             let mut vec_str = vec![start];
-            for j in 0..i {
-                vec_str.push(format!("{}", unwrap!(dims[j].size().as_int())));
+            for dim in dims.iter().take(i) {
```
Comment (Elarnon) [inline backend/mppa/src/printer.rs]: I see you ran clippy :)

Comment (nicoTolly) [reply backend/mppa/src/printer.rs]: Precisely :)

File: backend/mppa/src/context.rs (hunk 1 above)
Comment (nicoTolly) [reply backend/mppa/src/context.rs]: Oh yes, cargo fmt did that for me, I think

Comment (nicoTolly) [reply backend/mppa/src/context.rs]: No choice here, if I change that cargo fmt -- --check will fail... Is there any workaround ?

**Comment by nicoTolly** [comment]
bors r=Elarnon

**Pull request merged**